openwisp_controller/config/tests/test_admin.py

```diff
@@ -359,6 +359,65 @@ def test_template_not_contains_default_templates_js(self):
         response = self.client.get(path)
         self.assertNotContains(response, '// enable default templates')
 
+    def test_configuration_modified(self):
```

Comment:
wouldn't it be better to name it something like `test_configuration_templates_removed`?